src/modules/signer/wallet-connect-metamask.ts

```diff
@@ -38,14 +37,8 @@ export const createWalletConnectProvider = (
   );
   const walletConnectProvider = new Web3Provider({
     rpc,
-    connector: new Connector({ bridge, qrcodeModal: QRCodeModal }),
+    connector: new WalletConnect({ bridge, qrcodeModal: QRCodeModal }),
     infuraId,
   });
   return walletConnectProvider;
 };
-
-class Connector extends WalletConnect {
```

Comment:
`Connector` brings nothing new to `WalletConnect`

Reply:
Maybe this was left over from something related to https://github.com/energywebfoundation/iam-client-lib/commit/42ee4391701a66161d1a3a2c8cbac5e5aaf143d4#diff-25a6634263c1b1f6fc4697a04e2b9904ea4b042a89af59dc93ec1f5d44848a26 🤔